tests/evaluate/test_default_evaluator.py

```diff
@@ -2377,7 +2377,12 @@ def test_eval_results_table_json_can_be_prefixed_with_metric_prefix():
 
     client = mlflow.MlflowClient()
     artifacts = [a.path for a in client.list_artifacts(run.info.run_id)]
+
+    if metric_prefix is None:
+        metric_prefix = ""
+
     assert f"{metric_prefix}eval_results_table.json" in artifacts
+    assert results.artifacts[f"{metric_prefix}eval_results_table"].content
```

Comment:
Do we want to validate the contents of what is logged here as opposed to just validating that it's not None?